src/brackets.js

```diff
@@ -415,11 +415,11 @@ define(function (require, exports, module) {
             var node = e.target, url;
             while (node) {
                 if (node.tagName === "A") {
+                    e.preventDefault();
```

Comment:
We need to `preventDefault` before calling `NativeApp.openURLInDefaultBrowser(url);` due to some quirk in Electron.